src/App.tsx

```diff
@@ -20,12 +36,19 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => {
+                  setQuery(event.target.value);
+                }}
               />
             </div>
           </div>
         </div>
+        {
+          (query === '')
+            ? <MoviesList movies={moviesFromServer} />
+            : <MoviesList movies={filterMovies(query, moviesFromServer)} />
```

Comment:
I believe you can remove the ternary operator and always render component with filterMovies